nat-lab/tests/test_lana_validator.py

```diff
@@ -0,0 +1,48 @@
+from utils.analytics import Event, basic_validator
+
+TEST_EVENT = Event(
+    name="heartbeat",
+    category="service_quality",
+    datetime_local="2023-08-10T15:03:25.605768807+00:00",
+    external_links="86c4375d-014b:gamma:3,edcd3eea-7aca:beta:3",
+    connectivity_matrix="'1:2:3,0:2:2,0:1:3'",
+    fp="86bee206-9082",
+    members="gamma, beta",
+    connection_duration="0;11;11",
+    heartbeat_interval=3600,
+    received_data="0:0:0:0:0",
+    rtt="0:0:0:0:0",
+    sent_data="0:0:0:0:0",
+    nat_type="PortRestrictedCone",
+    mem_nat_types="Symmetric,PortRestrictedCone",
+)
+
+
+def test_lana_validators() -> None:
+    test_validator = (
```

Comment:
I would like to see beside this validator, individual validators of each type with good and bad events to cover as many cases as possible. In other words, increase granularity and the number of cases covered.
Ex: 
members_validator = EventValidator().add_members_validator(whatever)
assert members_validator.validate(good_event) == True
assert members_validator.validate(bad_event) == False

Reply:
I had made individual validators, but then changed it all as I thought that since the big one does cover all the cases then if this works then others would in principle also work.